contracts/deploy/001_core.js

```diff
@@ -16,8 +16,7 @@ const {
 const getStrategyGovernorAddress = async () => {
   const { governorAddr } = await hre.getNamedAccounts();
   if (isMainnet) {
-    // On Mainnet the governor is the TimeLock
-    return (await ethers.getContract("MinuteTimelock")).address;
+    return (await ethers.getContract("Governor")).address;
```

Comment:
nit: does not really matter since we'll never run mig 001 on mainnet but I think this is un-necessary. hre.getNamedAccount() should already return the proper address on mainnet given the config we have in hardhat.config.js

Reply:
Same comment in mig 005 and 006 below

Reply:
True, will clean it up